includes/Core/Modules/REST_Modules_Controller.php

```diff
@@ -0,0 +1,638 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Modules\REST_Modules_Controller
+ *
+ * @package   Google\Site_Kit\Core\Modules
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Modules;
+
+use Google\Site_Kit\Core\Modules\REST_Modules_Controller;
```

Comment:
I think this is not necessary here since this is the same place where the class is defined. Also, it triggers a duplicate symbol error in intelephense.

```suggestion

```

Reply:
Excellent catch @nfmohit ! Updated!